constraints/required.js

```diff
@@ -1,3 +1,23 @@
-export default function constraintsRequired(values/*, options*/) {
-  return (values.length > 0 && true) || false;
-}
+import { FORM } from '../namespaces';
+
+export default function constraintsRequired(values, options) {
+  if(values.length === 0){
+    return false;
+  }
+  else if(values.length === 1){
+    if(values[0].value?.length === 0){
```

Comment:
also: good that you check for empty string (a bit scared though for breaking current forms) 
But could you be consistent and do this in the else branch too then?